app/src/main/java/com/filipesperandio/simpletasks/MainActivity.kt

```diff
@@ -22,17 +22,9 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        tasks.presenterFactory(TaskPresenterFactory())
-
-        Observable.interval(1, TimeUnit.SECONDS)
-                .doOnNext { it + 1 }
-                .take(5)
-                .observeOn(AndroidSchedulers.mainThread())
-                .subscribe {
-                    val task = Task(it.toString(), false, it)
-                    tasksBus.onNext(task)
-                }
+            tasks.presenterFactory(TaskPresenterFactory())
```

Comment:
why the indentation changed here?

AndroidStudio has ways to auto-format the code for you. Look it up.

Keeping the code with the right style and format is crucial when working on complex diffs, so it is a good practice to keep it always clean. That means, indentation, line breaks, where `{} | () | ;` goes and code styles we can discuss later.